internal/cmd/commands/dev/dev.go

```diff
@@ -262,6 +269,40 @@ func (c *Command) Flags() *base.FlagSets {
 		Target: &c.flagContainerImage,
 		Usage:  `Specifies a container image to be utilized. Must be in <repo>:<tag> format`,
 	})
+	f.StringVar(&base.StringVar{
+		Name:       "event-format",
+		Target:     &c.flagEventFormat,
+		Completion: complete.PredictSet("cloudevents-json", "cloudevents-text"),
+		Usage:      `Event format. Supported values are "cloudevents-json" and "cloudevents-text".`,
+	})
+	f.StringVar(&base.StringVar{
+		Name:       "observations",
+		Target:     &c.flagObservations,
+		Completion: complete.PredictSet("true", "false"),
+		Usage:      `Emit observation events. Supported values are "true" and "false".`,
+	})
+	f.StringVar(&base.StringVar{
+		Name:       "audit",
+		Target:     &c.flagAudit,
+		Completion: complete.PredictSet("true", "false"),
+		Usage:      `Emit audit events. Supported values are "true" and "false".`,
+	})
+	f.StringVar(&base.StringVar{
+		Name:       "sys-events",
```

Comment:
It feels incongruous to have this flag have `-events` on the end but not the other two. Might I suggest `audit-events` and `observation-events`? That also then makes it clear that the filters will apply to those as well.

Another nit: If it's spelled out "observation-events" (and `observations` right now) it should probably be `system-events`.

Reply:
ty!